### DISCUSSION TITLE
Matrix optimizations

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#57] [Shipped - Live on production]
Feedback from different users:

Matrix functions not self-explanatory.

1. "Filter" Button doesn't explain the real function.
User idea: 
_Maybe rename to "Back" and another icon or better convert matrix to steps logic. Step 1 "Selection" (previously Filter) -> Button "Show matrix" -> Step 2 "Matrix_ (just like when importing personnel data)

2. Show more device information
Current: Only device type and device model
Interesting would be: Manufacturer, device type, device model and annex MPBetreibV (Anlage 1, Anlage 2, Anlage 1+2 oder Keine)

3. Add a legend to explain A(U) & H(V)

**Comment by moritzvolkmar** [comment]
@YvesR what do you think?

**Comment by YvesR** [comment]
1. stepper steals to much space, but we can change to "Adjust selection" and use a back icon
2. No space either, but we can allow click on device to open details, ok?
3. Need to check, alternative would also be mouse over, what you think?

**Comment by YvesR** [comment]
- September rollout adds information to matrix, click on device the device details
- Legend is coming next update
- Click on training will open the training details
